src/pages/Admin/BackgroundSelect/BackgroundSelect.scss

```diff
@@ -39,7 +39,7 @@
 
     &:hover {
       border: $border-radius--xxs solid $white;
-      padding-bottom: 60px - $border-radius--xxs;
+      padding-bottom: 50px - $border-radius--xxs;
```

Comment:
This 50px value and the one used a couple of lines above should be converted into a constant to avoid problems like this.